lib/rules/web/admin_console/4.4/check_links.xyaml

```diff
@@ -161,3 +161,44 @@ check_customized_oc_download_links:
     text: 下载oc命令行
     link_url: mirror.openshift.com/pub/openshift-v4/clients/oc
   action: check_link_and_text
+
+# --- links on navigator menu ---
+check_Administration_menu_for_normal_user:
```

Comment:
`check_administration_menu_for_normal_user` is ok, it's better if we can keep consistence with other rules although `check_Administration_menu_for_normal_user` looks good